_rules/css-restrict-orientation-b33eff.md

```diff
@@ -0,0 +1,224 @@
+---
+id: b33eff
+name: Page is not restricted to either `landscape` or `portrait` orientation using CSS transform property
+rule_type: atomic
+description: |
+  This rule checks that page content is not restricted to either `landscape` or `portrait` orientation using CSS transform property.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any element that is [visible](#visible) and has a CSS [transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) property with any of the below [transformation functions](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function)
+
+- [rotate](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)
+- [rotate3d](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate3d)
+- [rotateX](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateX)
+- [rotateY](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateY)
+- [rotateZ](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotateZ)
+- [matrix](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix)
+- [matrix3d](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix3d)
+
+that are applied conditionally on the [orientation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Media_features) with a value of `landsapce` or `portrait`.
+
+**Note:** These specific [transformation functions](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function) are of interest to this rule as they have the potential to affect the [rotation](https://drafts.csswg.org/css-transforms-2/#Rotate3dDefined) of a given element.
+
+## Expectation
+
+The target element has a [current transformation matrix](https://drafts.csswg.org/css-transforms/#current-transformation-matrix) that does not restrict the element to either `landspace` or `portrait` orientation.
```

Comment:
The expectation does not currently make sense. The current transformation matrix is  unrelated to the `landscape` and `portrait` orientations, which are allowed values of the `orientation` media feature. Once again, I'll bring up my suggested alternative:

> The target element is not rotated around the Z axis at an angle corresponding to +-90 degrees relative to the position of the element prior to transformation.

This could very well be put into a definition of what an "orientation lock" actually is. Regardless of what that definition ends up looking like, it has to be unambiguous. And also, the term "rotated" is not ambiguous; it is well defined in terms of a transformation matrix.